README.md

```
@@ -112,3 +112,139 @@ Komplette Historie siehe `CHANGELOG.md`.
 ## Beitrag & Feedback
 
 Pull Requests, Issues und Ideen sind willkommen. Bitte ASCII (ae/oe/ue) verwenden und Patches knapp kommentieren. Viel Erfolg mit dem Gesundheits-Logger!
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)